agoradatatools/etl/transform.py

```diff
@@ -151,21 +151,70 @@ def calculate_distribution(df: pd.DataFrame, col: str, is_scored, upper_bound) -
     return obj
 
 
+def count_grouped_total(df: pd.DataFrame,
```

Comment:
Nice! The other added benefit of this is the extra documentation which will also make developing in and testing this repo easier